### PR TITLE
Allow supabase to be configured through env var and expose release endpoint

### PR DESCRIPTION
**Pull request by mcandeia**

### CODE REVIEW

File: supabase.ts [hunk 1]
```diff
@@ -2,12 +2,16 @@ import { supabase } from "./deps.ts";
 
 let client: supabase.SupabaseClient | null = null;
 
+const SUPABASE_LIVE_ENDPOINT_ENV_VAR = "SUPABASE_LIVE_ENDPOINT";
+const SUPABASE_ELIVE_ANON_KEY_ENV_VAR = "SUPABASE_LIVE_ANON_KEY";
```
Comment (matheusgr) [inline supabase.ts:6]:
```suggestion
const SUPABASE_LIVE_ANON_KEY_ENV_VAR = "SUPABASE_LIVE_ANON_KEY";
```

File: supabase.ts [hunk 2]
```diff
@@ -2,12 +2,16 @@ import { supabase } from "./deps.ts";
...
 // From supabase docs:
 // "This key is safe to use in a browser if you have enabled Row Level Security for your tables and configured policies."
 export const SUPABASE_LIVE_ENDPOINT =
-  "https://ozksgdmyrqcxcwhnbepg.supabase.co";
+  Deno.env.get(SUPABASE_LIVE_ENDPOINT_ENV_VAR) ??
+    "https://ozksgdmyrqcxcwhnbepg.supabase.co";
 export const SUPABASE_LIVE_ANON_KEY =
-  "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Im96a3NnZG15cnFjeGN3aG5iZXBnIiwicm9sZSI6ImFub24iLCJpYXQiOjE2NTY3MjM3NDYsImV4cCI6MTk3MjI5OTc0Nn0.HMdsG6NZlq6dvYFCK1_tuJh38TmsqLeN8H4OktTHt_M";
+  Deno.env.get(SUPABASE_ELIVE_ANON_KEY_ENV_VAR) ??
```
Comment (matheusgr) [inline supabase.ts:13]:
```suggestion
  Deno.env.get(SUPABASE_LIVE_ANON_KEY_ENV_VAR) ??
```

**Review by matheusgr — APPROVED** [review]

File: routes/live/release.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { HandlerContext } from "$fresh/server.ts";
+import { LiveConfig } from "../../mod.ts";
+
+export const handler = async (
+  _req: Request,
+  ctx: HandlerContext<unknown, LiveConfig>,
+) => {
+  return new Response(
+    JSON.stringify(await ctx.state.release.state()),
```
Comment (guifromrio) [inline routes/live/release.ts:9]: 😍

File: supabase.ts [hunk 4]
```diff
@@ -2,12 +2,16 @@ import { supabase } from "./deps.ts";
...
+const SUPABASE_LIVE_ENDPOINT_ENV_VAR = "SUPABASE_LIVE_ENDPOINT";
+const SUPABASE_ELIVE_ANON_KEY_ENV_VAR = "SUPABASE_LIVE_ANON_KEY";
 // From supabase docs:
 // "This key is safe to use in a browser if you have enabled Row Level Security for your tables and configured policies."
 export const SUPABASE_LIVE_ENDPOINT =
-  "https://ozksgdmyrqcxcwhnbepg.supabase.co";
+  Deno.env.get(SUPABASE_LIVE_ENDPOINT_ENV_VAR) ??
+    "https://ozksgdmyrqcxcwhnbepg.supabase.co";
 export const SUPABASE_LIVE_ANON_KEY =
-  "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Im96a3NnZG15cnFjeGN3aG5iZXBnIiwicm9sZSI6ImFub24iLCJpYXQiOjE2NTY3MjM3NDYsImV4cCI6MTk3MjI5OTc0Nn0.HMdsG6NZlq6dvYFCK1_tuJh38TmsqLeN8H4OktTHt_M";
```
Comment (guifromrio) [inline supabase.ts:10]: Adeus hardcoded strings, foi bom enquanto durou

**Review by guifromrio — APPROVED** [review]

**Review by guifromrio — APPROVED** [review]

**Pull request merged**